docs/images.md

```diff
@@ -44,19 +45,69 @@ Google service account.  The service account on the instance
 will have a `https://www.googleapis.com/auth/devstorage.read_only`,
 so it can pull from the project's GCR, but not push.
 
-### Configuring Nodes to Authenticate to a Private Registry 
-Docker stores keys for private registries in a `.dockercfg` file.  Create a config file by running
-`docker login <registry>.<domain>` and then copy the resulting `.dockercfg` file to the root user's
-`$HOME` directory (e.g. `/root/.dockercfg`) on each node in the cluster.
+### Configuring Nodes to Authenticate to a Private Repository
+
+**Note for GKE:** if you are running on Google Container Engine (GKE), there will already be a `.dockercfg` on each node
+with credentials for Google Container Registry.  You cannot use this approach.
+
+Docker stores keys for private registries in the `$HOME/.dockercfg` file.  If you put this
+in the `$HOME` of `root` on a kubelet, then docker will use it.
+
+Here are the recommended steps to configuring your nodes to use a private registry.  In this
+example, run these on your desktop/laptop:
+   1. run `docker login [server]` for each set of credentials you want to use.
+   1. view `$HOME/.dockercfg` in an editor to ensure it contains just the credentials you want to use.
+   1. get a list of your nodes 
+      - for example: `nodes=$(kubectl get nodes -o template --template='{{range.items}}{{.metadata.name}} {{end}}')`
+   1. copy your local `.dockercfg` to the home directory of roon on each node.
+      - for example: `for n in $nodes; do scp ~/.dockercfg root@$n:/root/.dockercfg`
```

Comment:
You might make a note that for GCE this is only going to work temporarily. If a node gets replaced by the MIG or you scale up your cluster to add a new node then you need to manually repeat this step.